projects/find/find.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+import argparse
+
```

Comment:
here there should be an extra blank line (there should be 2 blank lines between top level objects)